### PR TITLE
Initial commit of GEO.COVER, GEO.COVER.RL

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/script/functions/GEOCOVER.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            while (decoder.next()) {
+              long location = decoder.getLocation();
+              if (GeoTimeSerie.NO_LOCATION != location) {
+                long cell = ((location & mask) >>> 4) | prefix;
+                cells.add(cell);
+              }
+            }            
+          }
+        }
+
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/script/functions/GEOCOVER.java]: ElementOrListStackFunction can work on any type so you still need to throw a WarpScriptException if the type of `element` is not what you expect.
So add a `else { throw ... }`.

File: warp10/src/main/java/io/warp10/script/functions/GEOCOVER.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+              long location = GTSHelper.locationAtIndex(gts, i);
+              if (GeoTimeSerie.NO_LOCATION != location) {
+                long cell = ((location & mask) >>> 4) | prefix;
+                cells.add(cell);
+              }
+            }            
+          }
+        } else if (element instanceof GTSEncoder) {          
+          if (rhumblines) {
+            throw new WarpScriptException(getName() + " hmm weird, we should never end here!");
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/script/functions/GEOCOVER.java]: So weird it can never happen, maybe change that to a comment?

**Comment by hbs** [comment]
PTAL

**Comment by ftence** [comment]
Looks good.

**Pull request merged**